src/vs/editor/contrib/quickAccess/browser/editorNavigationQuickAccess.ts

```diff
@@ -146,6 +147,10 @@ export abstract class AbstractEditorNavigationQuickAccessProvider implements IQu
 		if (!options.preserveFocus) {
 			editor.focus();
 		}
+		const model = editor.getModel();
+		if (model && 'getLineContent' in model) {
```

Comment:
@meganrogge why not use the existing method `getModel` here that does the cast for you?

https://github.com/microsoft/vscode/blob/ada796969b6599a66c6deb0d057aae4b4f3ecf4b/src/vs/editor/contrib/quickAccess/browser/editorNavigationQuickAccess.ts#L156-L160